Documentation/gettingstarted/istio.rst

```diff
@@ -140,464 +140,78 @@ Check the progress of the deployment (every service should have an
-- username
-- client IP address
-- timestamp
-
-To observe the Kafka messages sent by ``productpage``, we will run an
-additional ``authaudit-logger`` service.  This service fetches and
-prints out all messages from the ``authaudit`` Kafka topic.  Start
-this service:
-
-.. parsed-literal::
-
-    $ curl -s \ |SCM_WEB|\/examples/kubernetes-istio/authaudit-logger-v1.yaml | \\
-          istioctl kube-inject -f - | \\
-          kubectl apply --validate=false -f -
-    deployment "authaudit-logger-v1" created
-
-Check the progress of the deployment (every service should have an
-``AVAILABLE`` count of ``1``):
+- Set the `GATEWAY_URL`:
 
 ::
 
-    $ kubectl get deployments
-    NAME                  DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-    authaudit-logger-v1   1         1         1            1           20s
-    details-v1            1         1         1            1           22m
-    productpage-v2        1         1         1            1           4m
-    ratings-v1            1         1         1            1           19m
-    reviews-v1            1         1         1            1           22m
-    reviews-v2            1         1         1            1           19m
+   $ export GATEWAY_URL=$INGRESS_HOST:$INGRESS_PORT
 
-Check that the product REST API is still accessible, and that Cilium
-now denies at Layer-7 any access to the reviews and ratings REST API:
+- Confirn that the Bookinfo application is reachable from outside of the cluster:
 
 ::
 
-    $ export PRODUCTPAGE=`minikube service istio-ingressgateway -n istio-system --url | head -n 1`
-    $ for APIPATH in /api/v1/products /api/v1/products/0 /api/v1/products/0/reviews /api/v1/products/0/ratings; do echo ; curl -s -S "${PRODUCTPAGE}${APIPATH}" ; echo ; done
-
-    [{"descriptionHtml": "<a href=\"https://en.wikipedia.org/wiki/The_Comedy_of_Errors\">Wikipedia Summary</a>: The Comedy of Errors is one of <b>William Shakespeare's</b> early plays. It is his shortest and one of his most farcical comedies, with a major part of the humour coming from slapstick and mistaken identity, in addition to puns and word play.", "id": 0, "title": "The Comedy of Errors"}]
-
-    {"publisher": "PublisherA", "language": "English", "author": "William Shakespeare", "id": 0, "ISBN-10": "1234567890", "ISBN-13": "123-1234567890", "year": 1595, "type": "paperback", "pages": 200}
-
-    Access denied
-
-
-    Access denied
-
-This demonstrated that requests to the
-``/api/v1/products/<id>/reviews`` and
-``/api/v1/products/<id>/ratings`` URIs now result in Cilium returning
-``HTTP 403 Forbidden`` HTTP responses.
-
-Every login and logout on the product page will result in a line in
-this service's log. Note that you need to log in/out using the ``sign
-in``/``sign out`` element on the bookinfo web page. When you do, you
-can observe these kind of audit logs:
-
-::
-
-    $ export POD_LOGGER_V1=`kubectl get pods -l app=authaudit-logger,version=v1 -o jsonpath='{.items[0].metadata.name}'`
```

Comment:
What happened to the authaudit logger?

Reply:
Did not get that far yet & wanted to see how bad we want it in the GSG...

Reply:
@ianvernon I rebased the whole existing GSG to Istio 1.1.3, please take a look.